ai-ml/llm-multiple-gpus/falcon-40b/text-generation-inference.yaml

```
@@ -51,6 +51,9 @@ spec:
         volumeMounts:
           - mountPath: /dev/shm
             name: dshm
+          # mountPath is set to /data as it's the path where the HF_HOME environment
```

For images lower than TGI 2.3 the environment variable was `HUGGINGFACE_HUB_CACHE` instead of `HF_HOME`

```suggestion
          # mountPath is set to /data as it's the path where the HUGGINGFACE_HUB_CACHE environment
```